ophyd/device.py

```diff
@@ -1120,3 +1205,68 @@ def _repr_info(self):
 
         yield ('read_attrs', self.read_attrs)
         yield ('configuration_attrs', self.configuration_attrs)
+
+
+class _OphydAttrList(MutableSequence):
+    """list proxy to migrate away from Device.read_attrs and Device.config_attrs
+
+
+    """
+
+    def __init__(self, device, kind, remove_kind, recurse_key):
+        self._kind = kind
+        self._remove_kind = remove_kind
+        self._parent = device
+        self._recurse_key = recurse_key
+
+    def __internal_list(self):
```

Comment:
As of Friday, we thought it would, but in working with `read_attrs` and `configuration_attrs` we have come around to think that it's a nice convenience API on top of the distributed `kind` attributes. The localized source of truth is a better implementation (easier to get right and to debug) but for those times that you want to un-set or set a bunch of attributes at once, these lists are nice. We might keep them around too.

Reply:
I have moved a bit on removing these things, it is a pretty nifty API for setting a whole bunch of this state all at once.  I think we need to sit on this for a cycle or so before we make any deprecation decisions.